persistence/mongodb/migration-warning.include.md

```diff
@@ -0,0 +1 @@
+WARNING: Compatibility requires additional settings which must be configured otherwise data duplication and incorrect execution of business logic may occur.
```

Comment:
``` Suggestion
WARNING: Compatibility requires additional settings which must be configured, otherwise data duplication and incorrect execution of business logic may occur.
```